http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -150,7 +154,13 @@ class LeoPubsubMessageSubscriber[F[_]: Async: Timer: ContextShift: Concurrent](
                     // start cluster
                     _ <- runtimeConfig.cloudService.interpreter.startRuntime(StartRuntimeParams(resolvedCluster, now))
                     // clean-up info from follow-up table
-                    _ <- dbRef.inTransaction { followupQuery.delete(message.runtimeFollowupDetails) }
+                    _ <- dbRef.inTransaction {
+                      followupQuery.delete(message.runtimeFollowupDetails) >> clusterQuery.updateClusterStatus(
```

Comment:
I think we don't need to delete record from `followupQuery` anymore?

Reply:
Hm why not? I didn't change any behavior w.r.t. `followupQuery` here. I added the `updateClusterStatus` calls because `RuntimeService.start/stopRuntime` no longer updates the runtime status.

Reply:
I think the original reason why we need to do this clean up is we're doing `insert` when adding `followup` record, but I changed it to `upsert` at one point, and removed this clean up at one point...

Reply:
Ah gotcha.. I can remove this then